Makefile

```diff
@@ -0,0 +1,26 @@
+docker-compose-e2e = ./docker-compose.e2e.yml
+docker-compose-unit-tests = ./docker-compose.test.yml
+
+skip_markers ?=
+
+unit-tests: setup-unit-tests run-unit-tests
+
+setup-unit-tests:
+	docker-compose -f $(docker-compose-unit-tests) up -d --build
+	./scripts/wait-for-postgres.sh localhost 15432 test
+
+run-unit-tests:
+	$(eval markers := $(skip_markers) $(if $(skip_markers), and) not e2e)
+	docker-compose -f $(docker-compose-unit-tests) exec -T river pytest -m "$(markers)"
+
+e2e-tests: setup-e2e-tests run-e2e-tests
```

Comment:
I think we want two commands: one to run e2e tests locally and one to run tests against the `testy` host deployed on scaleway.

Reply:
Yep, we could have that once the testy env is ready 🙂